products/facilities/metadata.yaml

```diff
@@ -65,14 +45,14 @@ dataset_package:
       omit_columns: []
       columns: {}
   attachments:
-  - facilities_23v2_metadata.zip
+  - facilities_metadata.zip
```

Comment:
It's still an unanswered question how we deal with changing attachment or file names. Maybe there needs to be some templating. For now, I think it's best to just change the filename in our package.